src/components/steps/WalletDisclaimer.vue

```diff
@@ -42,7 +42,6 @@ export default {
       this.$router.push('/ftu/seed-type-selection')
     },
     nextStep() {
-      this.$store.dispatch('createMnemonics')
```

Comment:
Why do you delete the `createMnemonics` dispatch?

Reply:
I add createMnemonics before the component is created cause mnemonics were not shown when reloading the page.